example_classification_2.py

```diff
@@ -0,0 +1,118 @@
+"""Creates and compares classification models using sample clinical data.
+
+Please use this example to learn about healthcareai before moving on to the next example.
+
+If you have not installed healthcare.ai, refer to the instructions here:
+  http://healthcareai-py.readthedocs.io
+
+To run this example:
+  python3 example_classification_2.py
+
+This code uses the DiabetesClinicalSampleData.csv source file.
+"""
+import pandas as pd
+import sqlalchemy
+import sqlite3
+
+import healthcareai.trained_models.trained_supervised_model as tsm_plots
+from healthcareai.supvervised_model_trainer import SupervisedModelTrainer
+import healthcareai.common.file_io_utilities as io_utilities
+import healthcareai.common.write_predictions_to_database as hcaidb
+
+
+def main():
+    # Load data from a sample .csv file
+    prediction_dataframe = pd.read_csv('healthcareai/tests/fixtures/DiabetesClinicalSampleData.csv', na_values=['None'])
+
+    # Load data from a MSSQL server: Uncomment to pull data from MSSQL server
+    # server = 'localhost'
+    # database = 'SAM'
+    # query = """SELECT *
+    #             FROM [SAM].[dbo].[DiabetesClincialSampleData]
+    #             WHERE SystolicBPNBR is null"""
+    #
+    # engine = hcaidb.build_mssql_engine(server=server, database=database)
+    # prediction_dataframe = pd.read_sql(query, engine)
+
+    # Drop columns that won't help machine learning
+    columns_to_remove = ['PatientID']
+    prediction_dataframe.drop(columns_to_remove, axis=1, inplace=True)
+
+    # Load the saved model and print out the metrics
+    trained_model = io_utilities.load_saved_model('2017-05-31T12-36-21_classification_RandomForestClassifier.pkl')
```

Comment:
The 2017-05-31T12-36-21_classification_RandomForestClassifier.pkl file is not created when running classification example 1 (and no similar file is created either), classification example 2 fails at this point simply by not being able to load the trained model.